scenariolib/visit.go

```diff
@@ -114,7 +115,14 @@ func NewVisit(_searchtoken string, _uatoken string, _useragent string, language
 }
 
 func buildUserEmail(c *Config) string {
-	return fmt.Sprint(c.FirstNames[rand.Intn(len(c.FirstNames))], ".", c.LastNames[rand.Intn(len(c.LastNames))], c.Emails[rand.Intn(len(c.Emails))])
+	firstName := c.FirstNames[rand.Intn(len(c.FirstNames))]
+	lastName := c.LastNames[rand.Intn(len(c.LastNames))]
+	emailSuffix := c.Emails[rand.Intn(len(c.Emails))]
+	if lastName == "" {
```

Comment:
Why would lastName be empty?
Check the function to fill defaults
Unless you want to have a firstname@email.com?
Basically if you don't provide any of the firstname, lastname, email params, defaults will be added for you.
in defaults.go